tests/lexer.c

```diff
@@ -1,20 +1,17 @@
 #include "lexer.h"
 
 #include <unity.h>
+// NOLINTNEXTLINE(misc-include-cleaner,-warnings-as-errors)
 #include <unity_internals.h>
```

Comment:
@WannabeMO I believe line 5 can be removed since the unity test macros have been moved to `main.c`.

```suggestion
```